client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo/ReachContact.tsx

```diff
@@ -41,9 +42,10 @@ const ReachContact = (props: Props) => {
                         defaultValue={interactedContact.contactStatus}
                         render={(props) => {
                             const currentValue = getCurrentValue(props.value);
+                            const toDisableField = isFieldDisabled || currentValue.id === ContactStatusCodes.COMPLETED
```

Comment:
isn't isFieldDisabled already a check if the statusId equalling to completed status? (by the form's state) - why use it here?
what I'm asking here is when will there be a case where isFieldsDisabled is not equal to currentValue.id === ContactStatusCodes.COMPLETED?

Reply:
at the first time after the user changed the status to הושלם התחקור
this currentValue.id === ContactStatusCodes.COMPLETED return true
and isFieldDisabled returns false